admin/views/help_resource.php

```diff
@@ -7,100 +7,77 @@
 
 defined( 'ABSPATH' ) || die();
 
-$videos_data = [
-        [
-            'title' => __('How to Use the Plugin', 'social-polls-by-opinionstage'),
-            'title_link' => 'https://help.opinionstage.com/en/articles/2557183-how-to-use-the-wordpress-plugin/',
-            'video_id' => 'DMcosYCBFDs',
-        ],
-        [
-            'title' => __('How to Create a Quiz', 'social-polls-by-opinionstage'),
-            'video_id' => 'PPNIezl_wu0',
-        ],
-        [
-            'title' => __('How to Create a Poll', 'social-polls-by-opinionstage'),
-            'video_id' => 'xFUAwszhiuo',
-        ],
-        [
-            'title' => __('How to Create a Survey', 'social-polls-by-opinionstage'),
-            'video_id' => 'sUcGbCGwn_Q',
-        ],
-]
+$videos_data = array(
+	array(
+		'title'      => __( 'How to Use the Plugin', 'social-polls-by-opinionstage' ),
+		'title_link' => 'https://help.opinionstage.com/en/articles/2557183-how-to-use-the-wordpress-plugin/',
+		'video_id'   => 'DMcosYCBFDs',
+	),
+	array(
+		'title'    => __( 'How to Create a Quiz', 'social-polls-by-opinionstage' ),
+		'video_id' => 'PPNIezl_wu0',
+	),
+	array(
+		'title'    => __( 'How to Create a Poll', 'social-polls-by-opinionstage' ),
+		'video_id' => 'xFUAwszhiuo',
+	),
+	array(
+		'title'    => __( 'How to Create a Survey', 'social-polls-by-opinionstage' ),
+		'video_id' => 'sUcGbCGwn_Q',
+	),
+);
+
+
 ?>
 <div id="opinionstage-content">
-	<div class="opinionstage-header-wrapper">
-		<div class="opinionstage-logo-wrapper">
-			<div class="opinionstage-logo"></div>
-			<div class="opinionstage-connectivity-status"><?php echo esc_html( $os_options['email'] ); ?>
-				<form method="POST" action="<?php echo esc_url( get_admin_url( null, 'admin.php?page=' . OPINIONSTAGE_DISCONNECT_PAGE ) ); ?>" class="opinionstage-connect-form">
-					<button class="opinionstage-disconnect" type="submit">Disconnect</button>
-				</form>
+		<?php
+		$user_email = ! empty( $os_options['email'] ) ? $os_options['email'] : '';
+		require OPINIONSTAGE_PLUGIN_DIR . 'admin/template-parts/header-logo-line-logout-form.php';
+		if ( $os_client_logged_in ) {
+			?>
+		<div class="opinionstage-tutorials-and-help">
+			
+			<div class="opinionstage-tutorials-and-help__hero">
+				<div class="opinionstage-tutorials-and-help__container">
+					<h1 class="opinionstage-tutorials-and-help__hero__header"><?php esc_html_e( 'Tutorials & Help', 'social-polls-by-opinionstage' ); ?></h1>
+					
+					<div class="opinionstage-tutorials-and-help__buttons">
+						<a href="https://help.opinionstage.com" class=" opinionstage-button opinionstage-button__blue" target="_blank"><?php esc_html_e( 'Go To Help Center', 'social-polls-by-opinionstage' ); ?></a>
```

Comment:
all `target="_blank"` must have `rel="noopener"` as well (https://www.jitbit.com/alexblog/256-targetblank---the-most-underestimated-vulnerability-ever/)